product_sold_by_delivery_week/models/product_product.py

```diff
@@ -59,8 +59,8 @@ def _weekly_sold_delivered_domain(self, date_start, date_end):
             ("date_done", "<", date_end),
             ("picking_type_id", "in", picking_types.ids),
         ]
-        if partner_id:
-            picking_domain += [("partner_id", "child_of", partner_id)]
+        if partner_ids:
+            picking_domain += [("partner_id", "in", partner_ids)]
```

Comment:
We must analyze the best option...
```suggestion
            picking_domain += [("partner_id", "child_of", partner_ids)]
```